### DISCUSSION TITLE
Error on inserting into auth_logins

### DISCUSSION THREAD

**Discussion by aneesha** [codeigniter4/shield#1108] [Q&A]
Hi

I get an error "CodeIgniter\Database\MySQLi\Connection::_escapeString(): Argument #1 ($str) must be of type string, CodeIgniter\I18n\Time given" which is from line 77 in VENDORPATH/codeigniter4/shield/src/Models/LoginModel.php : 77 

77         $return = $this->insert([
78             'ip_address' => $ipAddress,
79             'user_agent' => $userAgent,
80             'id_type'    => $idType,
81             'identifier' => $identifier,
82             'user_id'    => $userId,
83             'date'       => Time::now(),
84             'success'    => (int) $success,

Any assistance would be appreciated.

Thanks

**Comment by datamweb** [comment]
Hi, 
Use the latest version of Codeigniter and the latest version of Shield.
